src/extensions/default/HealthData/htmlContent/healthdata-notification-dialog.html

```diff
@@ -0,0 +1,16 @@
+<div class="health-data-notification modal">
```

Comment:
@MarcelGerber yup, agreed. I sent a note to our legal/privacy team with an alternative proposal that provides a toast notification in the upper right hand corner instead of the modal dialog at first launch.

It's still very up front and semi-in-your-face, but I think it's a little bit more elegant way to introduce the concept.